services/brig/src/Brig/Team/API.hs

```diff
@@ -156,6 +157,17 @@ routesPublic = do
     Doc.response 404 "No pending invitations exists." Doc.end
     Doc.response 409 "Multiple conflicting invitations to different teams exists." Doc.end
 
+  get "/teams/:tid/size" (continue teamSizePublicH) $
+    accept "application" "json"
+      .&. header "Z-User"
+      .&. capture "tid"
+
+  document "GET" "teamSize" $ do
+    Doc.summary "Returns the number of team members as an integer."
+    Doc.returns (Doc.ref Public.modelTeamSize)
+    Doc.response 200 "Invitation successful." Doc.end
+    Doc.response 403 "No permission (not admin or owner of this team)." Doc.end
```

Comment:
Can you please add a note about this endpoint returning potentially stale size as it depends on `refresh_interval` of the index which is in seconds (5 seconds in prod, iirc).